std/src/main/scala/algebra/std/set.scala

```diff
@@ -6,12 +6,15 @@ import algebra.ring.Semiring
 package object set extends SetInstances
 
 trait SetInstances {
-  implicit def setEq[A]: Eq[Set[A]] = new SetEq[A]
+  implicit def setPartialOrder[A]: PartialOrder[Set[A]] = new SetPartialOrder[A]
   implicit def setSemiring[A] = new SetSemiring[A]
 }
 
-class SetEq[A] extends Eq[Set[A]] {
-  def eqv(x: Set[A], y: Set[A]): Boolean = x == y
+class SetPartialOrder[A] extends PartialOrder[Set[A]] {
```

Comment:
might want to override eqv so it will not have to do two subset checks when they are not equal.